docs/installation.rst

```diff
@@ -9,74 +9,68 @@ Requirements
 
 sunpy requires Python 3.8 or higher.
 
-Installing Scientific Python and sunpy
-======================================
+Installing Scientific Python and ``sunpy``
+==========================================
 
-sunpy is part of the wider ecosystem of scientific Python packages for solar physics.
-Therefore a working sunpy installation is more about installing the scientific Python ecosystem than sunpy itself.
+``sunpy`` is part of the wider ecosystem of scientific Python packages for solar physics.
+Therefore a working ``sunpy`` installation is more about installing the scientific Python ecosystem than ``sunpy`` itself.
 
-If you do not currently have a working scientific Python distribution this guide will set you up with the Miniconda, which makes it easy to install and manage your scientific Python packages.
+If you do not currently have a working scientific Python distribution this guide will set you up with the Miniforge, which makes it easy to install and manage your scientific Python packages.
 
-To install the Miniconda Python distribution follow the instructions at
-`here <https://docs.conda.io/en/latest/miniconda.html>`__.
-Although Miniconda makes it simple to switch between Python versions, we recommend that new users install the latest Python 3.x version of Miniconda.
+`To install the Miniforge Python distribution follow these instructions <https://github.com/conda-forge/miniforge#install>`__.
 
-The reason we choose Miniconda over Anaconda, is mainly due to the size as Anaconda comes with a full install of packages you probably do not need and this way you have more direct control over what has been installed into your Python virtual environment.
-Furthermore, you bypass the need for the conda resolver to sort out your root environment which should make conda faster to use.
+We do not recommend that you use Anaconda, there are known conflicts with the "defaults" channel and the "conda-forge" channel which has the ``sunpy`` package.
 
-Installing sunpy using Miniconda
---------------------------------
+Installing ``sunpy`` using Miniforge
+------------------------------------
 
-To install sunpy launch a system command prompt or the 'Anaconda Prompt' (under Windows).
-First configure conda for to add the `conda-forge channel <https://conda-forge.org/>`__::
+To install ``sunpy``, launch a system command prompt or terminal.
```

Comment:
Does miniforge come with a windows prompt thing?

Reply:
I need to install it to check